lib/event-bus.js

```diff
@@ -1,3 +1,5 @@
+'use strict'
```

Comment:
Why is this here?
Why isn't it using the class syntax?

Reply:
this is commonjs so you should use strict mode. In esm you have always strict mode.
By using the prototypical inheritance we avoid the if condition in the function.